internal/logs/traces/types.go

```diff
@@ -0,0 +1,34 @@
+package traces
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3/trace"
+)
+
+type endpoints []trace.EndpointInfo
+
+func (es endpoints) String() string {
+	bldr := strings.Builder{}
+	bldr.WriteByte('[')
+	for i, e := range es {
+		bldr.WriteString(e.String())
```

Comment:
Need to think on how we marshal internal structures. For example, do we need to json-escape this string (given that any JSON encoder in logger adapter will also escape it, resulting in double escaping like `","` -> `"\,"` -> `"\\\,"`)?

Just using `json.Marshal(es)` does not work, unfortunately. json package does not use Stringer's `String` method for marshaling.

Reply:
we can implement https://pkg.go.dev/encoding#TextMarshaler for the endpoint.info - it will used by json.Marshal.